components/support/test/src/main/java/mozilla/components/support/test/espresso/Matchers.kt

```diff
@@ -0,0 +1,38 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.support.test.espresso
+
+import android.view.View
+import org.hamcrest.CoreMatchers.not
+import org.hamcrest.Matcher
+import android.support.test.espresso.matcher.ViewMatchers.isChecked as espressoIsChecked
+import android.support.test.espresso.matcher.ViewMatchers.isDisplayed as espressoIsDisplayed
+import android.support.test.espresso.matcher.ViewMatchers.isEnabled as espressoIsEnabled
+import android.support.test.espresso.matcher.ViewMatchers.isSelected as espressoIsSelected
```

Comment:
I believe the issue is that your importing **android.support** and should import from **androidx.test**
```kotlin
import androidx.test.espresso.matcher.ViewMatchers.isChecked as espressoIsChecked
import androidx.test.espresso.matcher.ViewMatchers.isDisplayed as espressoIsDisplayed
import androidx.test.espresso.matcher.ViewMatchers.isEnabled as espressoIsEnabled
import androidx.test.espresso.matcher.ViewMatchers.isSelected as espressoIsSelected

```

In your[ initial implementation](https://github.com/mozilla-mobile/firefox-tv/blob/6b36f79c030b186feba68562fb999415031cae68/app/src/androidTest/java/org/mozilla/tv/firefox/helpers/Matchers.kt#L17) you wasn't using **androidx** but here you are.

Reply:
Built it locally and it worked.